tests/jest.config.dom.js

```diff
@@ -3,6 +3,8 @@ const baseConfig = require('kcd-scripts/jest')
 
 module.exports = {
   ...baseConfig,
+  moduleFileExtensions: [...baseConfig.moduleFileExtensions, 'd.ts'],
```

Comment:
Why does jest need `.d.ts` files? They shouldn't have any runtime behavior.

Reply:
I don't why but without using .d.ts jest not recognizes those files says that the module can"t be found

Reply:
Sounds like it's resolving a .d.ts file which it shouldn't do. Could you be a bit more specific where this exactly failing?

Reply:
I have removed both lines, Yesterday jest gave some problems  🤷🏼

Reply:
It's Friday so `jest` is like :shrug: